src/containers/CodeReminderModal/CodeReminderModal.test.jsx

```diff
@@ -153,6 +161,26 @@ describe('CodeReminderModalWrapper', () => {
     expect(spy).toHaveBeenCalledWith(couponId, codeReminderRequestData(2));
   });
 
+  it('returns the correct data if learner portal is not enabled', () => {
+    spy = jest.spyOn(EcommerceApiService, 'sendCodeReminder');
+    const codeRemindData = [data, data];
+    const wrapper = mount(<CodeReminderModalWrapper
+      data={{
+        ...codeRemindData,
+        selectedCodes: codeRemindData,
+        portalConfiguration: { ...codeRemindData.portalConfiguration, enableLearnerPortal: false },
+      }}
+      isBulkRemind
+    />);
+
+    expect(wrapper.find('.bulk-selected-codes').text()).toEqual('Selected Codes: 2');
+    expect(wrapper.find('#email-template')).toBeTruthy();
+    wrapper.find('.modal-footer .code-remind-save-btn').hostNodes().simulate('click');
+    const expectedData = codeReminderRequestData(2);
+    delete expectedData.base_enterprise_url;
+    expect(spy).toHaveBeenCalledWith(couponId, codeReminderRequestData(2));
```

Comment:
This test confuses me. Do we use expectedData?  I would have expected the test to have been called with expectedData, not a fresh call to the function

Reply:
It should be called with `expectedData`, and in fact should probably not be passing. Good catch!